app/controllers/application_controller.rb

```diff
@@ -27,6 +29,37 @@ class ApplicationController < ActionController::Base
 
   helper_method :content_item, :content_item_hash, :publication
 
+  def self.force_html_format(only: nil, except: nil)
```

Comment:
I wonder if it would make sense to extract it into a gem, so we could use it in other apps as well.